docs/releases/1.2.txt

```diff
@@ -309,7 +309,7 @@ Django's :doc:`internationalization framework </topics/i18n/index>` has been exp
 with locale-aware formatting and form processing. That means, if enabled, dates
 and numbers on templates will be displayed using the format specified for the
 current locale. Django will also use localized formats when parsing data in
-forms. See :ref:`Format localization <format-localization>` for more details.
+forms. See :doc:`Format localization </topics/i18n/formatting>` for more details.
```

Comment:
As above, since the title of /topics/i18n/formatting is "Format localization" it's redundant to specify it again.